LICENSE.md

```diff
@@ -22,3 +22,31 @@ Folders containing files under different permissive license than Apache 2.0 are
 - [TESTS/mbed_hal/trng/pithy](./TESTS/mbed_hal/trng/pithy) - BSD-3-Clause
 - [tools/data/rpc](./tools/data/rpc) - MIT
 - [targets](./targets) - PBL, BSD-style, MIT-style, Zlib-style, Public-domain
+
+The Python modules used by Mbed tools are used under the following licenses:
+
+- [colorama](https://pypi.org/project/colorama) - BSD-3-Clause
+- [urllib3](https://pypi.org/project/urllib3) - MIT
+   - [secure](https://pypi.org/project/secure) - MIT
+- [prettytable](https://pypi.org/project/prettytable) - BSD-3-Clause
+- [junit-xml](https://pypi.org/project/junit-xml) - MIT
+- [pyyaml](https://pypi.org/project/pyyaml) - MIT
+- [jsonschema](https://pypi.org/project/jsonschema) - MIT
+- [future](https://pypi.org/project/future) - MIT
+- [six](https://pypi.org/project/six) - MIT
+- [mbed-cloud-sdk](https://pypi.org/project/mbed-cloud-sdk) - Apache 2.0
+- [requests](https://pypi.org/project/requests) - Apache-2.0
+- [idna](https://pypi.org/project/idna) - **TODO**, *BSD-like / BSD-style ?*
```

Comment:
```suggestion
- [idna](https://pypi.org/project/idna) - BSD-3-Clause
```

Appears to be the same: https://opensource.org/licenses/BSD-3-Clause
https://github.com/kjd/idna/blob/master/LICENSE.rst